### ISSUE TITLE
Aligning OBO Dashboard with OBO Community Health

### ISSUE THREAD

**Issue by lschriml** [cthoyt/obo-community-health#12]
Hello Charlie,

The groups discussions yesterday got me thinking of how we could incorporate the passing of dashboard checks into the health report. 

I have a simple proposal, the [OBO dashboard](https://dashboard.obofoundry.org/) has 13 checks aligned to [OBO Principles](https://obofoundry.org/principles/fp-000-summary.html). I suggest that a Dashboard score base on the number of green check market (13), minus the number of red X's. This will enable us to see how ontologies improve in passing the Dashboard checks. 

What do you think ? 
Cheers,
Lynn

**Comment by cthoyt** [comment]
Not sure what group discussions you're referring to. Perhaps one of the closed OBO Foundry working group meetings?

I don't think I understand what you're suggesting. Is it that you want me to import the metrics from the OBO Dashboard and factor those into the OBO community health score?

If that's the case, I'm not sure if that's completely appropriate. Some things would then be duplicated (e.g., checking a valid license) and some things are irrelevant for the "community health" for each ontology (e.g., making sure all terms have definitions). 

I've defined the scope pretty narrowly in https://github.com/cthoyt/obo-community-health#why-does-this-exist but I'd be willing to expand it. It's not my goal to subsume or replace the OBO Dashboard with this project, and like I noted in the README, the report generated by this repository is currently a reflection of my personal values.

Additionally, I'm cautious to import analysis from OBO Dashboard (or any other tool that I didn't write myself) because I don't understand its code. Further, because it's an OBO Foundry project, there's a lot of red tape to make any changes that I might want. Usually my solution to get around this kind of thing is to re-implement tools I want from scratch, and this would take a lot of time and effort.

**Comment by matentzn** [comment]
I will tell you exactly - no need to check anything at all, nor to calculate - there is a score already and its published in a stable JSON file. We can deal with that once we have community wider agreement on the formula. Will give more details later.

**Comment by lschriml** [comment]
Hello Charlie,
   Sorry, I was referring to the conversation during the OBO Operations call yesterday. 
   I thought you were on the call. 

 I was just suggesting to add a score that is a count of the passed/not passed Dashboard checks to the Health report. 
 If an ontology had all green checkmarks, no red X's, it would have a score of 13. 
 If the ontology had only one red X, a score of 12. 

 I understand if that is out of scope of the health report.

 Just one more thought, another way to look at contributors, is the number of GitHub users creating ticket (issue author).
 I wasn't sure if the number of contributors was defined by the number of people who do commits to the repository.

 I appreciate your considering these ideas.
 
 Cheers,
  Lynn

**Comment by matentzn** [comment]
Blocked by https://github.com/OBOFoundry/OBO-Dashboard/issues/26
